light-zk.js/src/types/programParameters.ts

```diff
@@ -4,4 +4,5 @@ export type ProgramParameters = {
   verifierIdl: Idl;
   inputs: any; // object of proof and other inputs
   path: string;
+  accounts?: any; // object of accounts
```

Comment:
why not replace `any` with the type?

Reply:
We don't know what accounts are added in a psp before hand. The type needs to be generic. In the future we should make it depend on the idl.